src/app/vehicle-technical-record/vehicle-technical-record.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { VehicleTechnicalRecordComponent } from './vehicle-technical-record.component';
+
+describe('VehicleTechnicalRecordComponent', () => {
+  let component: VehicleTechnicalRecordComponent;
+  let fixture: ComponentFixture<VehicleTechnicalRecordComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ VehicleTechnicalRecordComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(VehicleTechnicalRecordComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Can add more tests?

Reply:
The Vehicle Tech Record page / component is just a placeholder at this stage to get us to the Test Record - the Tech Records team will be replacing, re-working or adding to this component in upcoming sprints, and, I'm sure, will add more tests as appropriate  👍🏻